.travis.yml

```diff
@@ -1,4 +1,4 @@
-langauge: ruby
+language: ruby #language was spelled incorrectly here.
```

Comment:
This PR looks good! 
One minor thing though: this comment is rather unnecessary.